lib/recurly/invoice.php

```diff
@@ -74,6 +74,45 @@ public function invoiceNumberWithPrefix() {
     return $this->invoice_number_prefix . $this->invoice_number;
   }
 
+  /**
+   * Refunds an open amount from the invoice and returns a new refund invoice
+   * @param Integer amount in cents to refund from this invoice
+   * @return Recurly_Invoice a new refund invoice
+   */
+  public function refundAmount($amount_in_cents) {
+    $doc = $this->createDocument();
+
+    $root = $doc->appendChild($doc->createElement($this->getNodeName()));
+    $root->appendChild($doc->createElement('amount_in_cents', $amount_in_cents));
+
+    return $this->createRefundInvoice($this->renderXML($doc));
+  }
+
+  /**
+   * Refunds given line items from an invoice and returns new refund invoice
+   * @param Array line items to refund (see 'REFUND ATTRIBUTES' for schema)
```

Comment:
Where do I find out about REFUND ATTRIBUTES? Seems like I should be able to pass instances of `Recurly_Adjustment` in here.